pkgs/development/tools/twitch-cli/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "twitch";
+  version = "1.1.19";
+
+  src = fetchFromGitHub {
+    owner = "twitchdev";
+    repo = "twitch-cli";
+    rev = "v${version}";
+    hash = "sha256-osR7+oQ0QBIk/OrMb4Txhb31zELi+eoM47GsWgF/PAg=";
+  };
+
+  vendorHash = "sha256-OhcRMXY8s+XciF+gV3cZ8fxtzo9+I76tBPZ0xG8ddHU=";
+
+  doCheck = false;
```

Comment:
Add a comment on why the checks are disabled.

Reply:
```
=== RUN   TestApiRequest
    test_setup.go:17: 
                Error Trace:    test_setup.go:17
                                                        api_request_test.go:15
                Error:          Expected nil, but got: &fs.PathError{Op:"mkdir", Path:"/homeless-shelter", Err:0xd}
                Test:           TestApiRequest
Error reading body: Get "potato": unsupported protocol scheme ""--- FAIL: TestApiRequest (0.00s)
=== RUN   TestNewRequest
    test_setup.go:17: 
                Error Trace:    test_setup.go:17
                                                        api_test.go:23
                Error:          Expected nil, but got: &fs.PathError{Op:"mkdir", Path:"/homeless-shelter", Err:0xd}
                Test:           TestNewRequest
{
  "data": []
}
{"data":[]}
{"data":[],"pagination":{"cursor":"test"}}
Endpoint responded with status 204
{"data":[]}
--- FAIL: TestNewRequest (0.00s)
=== RUN   TestValidOptions
    test_setup.go:17: 
                Error Trace:    test_setup.go:17
                                                        api_test.go:73
                Error:          Expected nil, but got: &fs.PathError{Op:"mkdir", Path:"/homeless-shelter", Err:0xd}
                Test:           TestValidOptions
--- FAIL: TestValidOptions (0.00s)
=== RUN   TestGetClientInformation
    test_setup.go:17: 
                Error Trace:    test_setup.go:17
                                                        api_test.go:83
                Error:          Expected nil, but got: &fs.PathError{Op:"mkdir", Path:"/homeless-shelter", Err:0xd}
                Test:           TestGetClientInformation
--- FAIL: TestGetClientInformation (0.00s)
=== RUN   TestNewClient
    test_setup.go:17: 
                Error Trace:    test_setup.go:17
                                                        client_test.go:17
                Error:          Expected nil, but got: &fs.PathError{Op:"mkdir", Path:"/homeless-shelter", Err:0xd}
                Test:           TestNewClient
--- FAIL: TestNewClient (0.00s)
```

Reply:
Probably requires networking then, comment that.